modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -159,11 +179,14 @@ private var urlSessionStore = SynchronizedDictionary<String, URLSession>()
                 onProgressReady?(dataTask.progress)
             }
 
+            taskIdentifier = dataTask.taskIdentifier
+            challengeHandlerStore[dataTask.taskIdentifier] = taskDidReceiveChallenge
```

Comment:
@DavidHorvath we also need to store the credential associated with the task identifier here
```
credentialStore[dataTask.taskIdentifier] = credential
```